### ISSUE TITLE
Calendar not working... asking for password when option disabled...

### ISSUE THREAD

**Issue by xios01** [SickRage/SickRage#3879]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

SickRage Info:
Branch: master
Commit: 96b53dee713dc35ebf295d0f37a67b2a36fb4ea3
Version: v2017.06.05-1
Database Version: 44.0
  Python Version: 2.7.13 (default, Apr 20 2017, 12:13:37) [GCC 6.3.0]
  SSL Version: LibreSSL 2.5.4
  OS: Linux-3.16.0-4-amd64-x86_64-with
  Locale: None.None
```
PASTE LOGS HERE
```
First, This had been working flawlessly... I did have to rebuild my server a while back, and this time loaded sickrage up via docker ... all but the calendar works great... 

Logs show nothing of interest, even with debug... no extra verbiage comes in when I attempt to retrieve the calendar via webcal... not even a login attempt.. 

Basically, I try to add the webcal://my.url.com:8080/calendar/  and it will not add to google calendar...  it fails...

I then tried with https://icalendar.org/validator.html to see what was going on it also failed... and upon examining the source... the source showed the password screen... 

I verified that "Unprotected calendar" was checked... I even unchecked it, restarted sickrage, checked it again, restarted and verified... same result...

**Comment by sharkykh** [comment]
@xios01 This is silly. But have you tried the URL **without** the trailing slash? there seems to be a small typo in the current code that doesn't allow for trailing slashes on the calendar route.
